src/op_analytics/datapipeline/models/templates/transaction_fees.sql.j2

```diff
@@ -16,36 +16,51 @@ pt1 AS (
     t.dt,
     t.chain,
     t.chain_id,
+    t.network,
     t.nonce,
+    t.transaction_index,
     t.from_address,
     t.to_address,
     t.block_number,
     t.block_timestamp,
+    -- Transaction Attributes
+    t.transaction_type,
     t.gas_price,
+    t.gas AS gas_limit,
     t.receipt_gas_used,
     t.receipt_l1_gas_used,
-    -- Fees
-    t.receipt_l1_fee AS l1_fee,
+    -- Gas
     t.receipt_l1_gas_price,
     t.receipt_l1_blob_base_fee,
+    b.base_fee_per_gas,
+    t.max_priority_fee_per_gas,
+    -- Fees
+    hexstr_bytelen(t.input) AS input_byte_length,
+    hexstr_calldata_gas(t.input) AS input_calldata_gas,
+    t.transaction_type = 126 AS is_system_transaction,
+    ( -- Attributes Deposited Spec https://specs.optimism.io/protocol/deposits.html#l1-attributes-deposited-transaction
+      t.transaction_type = 126
+      AND t.from_address = lower('0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001')
+      AND t.to_address = lower('0x4200000000000000000000000000000000000015')
+    ) AS is_attributes_deposited_transaction,
+    CASE WHEN t.gas_price = 0 THEN 0 ELSE t.receipt_l1_fee END AS l1_fee,
+    if(t.max_priority_fee_per_gas = 0, t.gas_price - t.max_priority_fee_per_gas, 0) AS base_legacy_fee_per_gas,
```

Comment:
wasn't clear to me why do we subtract `max_priority_fee_per_gas` when it is equal to 0.  That made me think that the logic is possible backwards.  @MSilb7  can you comment here for posterity?